app/helpers/compute_resources_vms_helper.rb

```diff
@@ -237,13 +237,23 @@ def vm_host_action(vm)
 
   def vm_import_action(vm, html_options = {})
     return unless Host.for_vm(@compute_resource, vm).empty?
-    display_link_if_authorized(
-      _("Import"),
+    link1 = display_link_if_authorized(
+      _("Import as managed"),
```

Comment:
`Import as managed Host`?
I'm wondering if users understand the difference between managed and unmanaged.

Reply:
Hold on with this, I'm working on a blog post which I'd like to convert to manual update later. I wouldn't change it here since the only notion users can see is the Manage / Unmanage button in host form.